pkgs/development/tools/azure-static-sites-client/default.nix

```
@@ -0,0 +1,116 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, curl
+, icu70
+, libkrb5
+, lttng-ust
+, openssl_1_1
+, zlib
+, azure-static-sites-client
+  # "latest", "stable" or "backup"
+, versionFlavor ? "stable"
+}:
+let
+  icu = icu70;
+  versions = lib.importJSON ./versions.json;
+  flavor = with lib; head (filter (x: x.version == versionFlavor) versions);
+  fetchBinary = runtimeId: fetchurl {
+    url = flavor.files.${runtimeId}.url;
+    sha256 = flavor.files.${runtimeId}.sha;
+  };
+  sources = {
+    "x86_64-linux" = fetchBinary "linux-x64";
+    "x86_64-darwin" = fetchBinary "macOS";
+  };
+  isOldVersion = lib.versionOlder flavor.buildId "1.0.021611";
+in
+stdenv.mkDerivation rec {
+  name = "${pname}-${versionFlavor}-${version}";
+  pname = "StaticSitesClient";
+  version = flavor.buildId;
+
+  src = sources.${stdenv.targetPlatform.system};
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ] ++ lib.optionals isOldVersion [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    curl
+    icu
+    openssl_1_1
+    libkrb5
+    lttng-ust
+    stdenv.cc.cc.lib
+    zlib
+  ];
+
+  dontUnpack = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install --mode 755 "$src" -D "$out/bin/${pname}"
+
+    declare icu_major_v="${with lib; head (splitVersion (getVersion icu.name))}"
+    for icu_lib in 'icui18n' 'icuuc' 'icudata'; do
+      patchelf --add-needed "lib''${icu_lib}.so.$icu_major_v" "$out/bin/${pname}"
+    done
```

```suggestion
    for icu_lib in 'icui18n' 'icuuc' 'icudata'; do
      patchelf --add-needed "lib''${icu_lib}.so.${with lib; head (splitVersion (getVersion icu.name))}" "$out/bin/${pname}"
    done
```
if we already have a nix variable we should not convert it into a bash one. If the line is to long or complex we can move it in to a let in.

---

Done.